# SPDX-License-Identifier: BSD-2-Clause
# Copyright (c) 2026, fsimg contributors
# Select the existing native interfaces on 64-bit FreeBSD as well as macOS.
--- a/src/fatcat.cpp
+++ b/src/fatcat.cpp
@@ -3 +3 @@
-#ifdef __APPLE__
+#if defined(__APPLE__) || defined(__FreeBSD__)
--- a/src/core/FatSystem.h
+++ b/src/core/FatSystem.h
@@ -12 +12 @@
-#ifdef __APPLE__
+#if defined(__APPLE__) || defined(__FreeBSD__)
